django/contrib/gis/db/backends/mysql/operations.py

```diff
@@ -100,7 +100,12 @@ def get_geometry_converter(self, expression):
         srid = expression.output_field.srid
         if srid == -1:
             srid = None
+        cls = expression.output_field.geom_class
```

Comment:
I'm not sure if `cls` is a good name since that could be confused with the classmethod argument naming convention. Would something like `output_cls` be an improvement?